controls/V-93475.rb

```diff
@@ -53,9 +53,11 @@
 
     application_accounts = input('application_accounts_local')
 
+    excluded_accounts = input('excluded_accounts_local')
```

Comment:
Did we add this to the shared inputs in the `inspec.yml` file? We want to make sure that we have a default value set there to ensure stability.

Reply:
Inspec.yml already has it added:
https://github.com/mitre/microsoft-windows-server-2019-stig-baseline/blob/master/inspec.yml#L330